test/cpp/api/modules.cpp

```diff
@@ -979,6 +979,20 @@ TEST_F(ModulesTest, HingeEmbeddingLoss) {
   ASSERT_EQ(input.sizes(), input.grad().sizes());
 }
 
+TEST_F(ModulesTest, MultiMarginLoss) {
+  auto weight = torch::tensor({0.3, 0.3, 0.4}, torch::kFloat);
+  MultiMarginLoss loss(MultiMarginLossOptions().margin(2).weight(weight));
+  auto input = torch::tensor({{0.2, 0.2, 0.6}, {0.1, 0.8, 0.1}, {0.9, 0.09, 0.01}}, torch::requires_grad());
+  auto target = torch::tensor({2, 1, 0}, torch::kLong);
+  auto output = loss->forward(input, target);
+  auto expected = torch::tensor({0.3056}, torch::kFloat);
+  auto s = output.sum();
+  s.backward();
+
+  ASSERT_TRUE(output.allclose(expected, 1e-04));
```

Comment:
It seems that this assert is failing on CI:
```
Oct 08 13:47:16 [ RUN      ] ModulesTest.MultiMarginLoss
Oct 08 13:47:16 /var/lib/jenkins/workspace/test/cpp/api/modules.cpp:992: Failure
Oct 08 13:47:16 Value of: output.allclose(expected, 1e-04)
Oct 08 13:47:16   Actual: false
Oct 08 13:47:16 Expected: true
Oct 08 13:47:16 [  FAILED  ] ModulesTest.MultiMarginLoss (0 ms)
```
The easiest way to debug is likely to run locally and see what causes the difference in output vs. expected value